File: 3189-find-champion-ii/find-champion-ii.cpp
class Solution {
public:
    int findChampion(int n, vector<vector<int>>& edges) {
        
        vector<int> num(n,0);

        for(auto&i: edges)
        {
            num[i[1]]=1;
        }
        if (accumulate(num.begin(),num.end(),0)<n-1)
            return -1;
        else 
        return distance(num.begin(),find(num.begin(),num.end(),0));
        
    }
};

File: 3189-find-champion-ii/find-champion-ii.py
class Solution:
    def findChampion(self, n: int, edges: List[List[int]]) -> int:
        arr=[0]*n
        for vertex in edges:
            arr[vertex[1]]=1
        if arr.count(0)>1:
            return -1
        else :
            return arr.index(0)
